### PR TITLE
Script Overview: fix notification flash (again)

### PR DESCRIPTION
**Pull request by Erin007**
[LP-145](https://codedotorg.atlassian.net/browse/LP-145) announcement appears briefly on the script overview page when it shouldn't

The verified resources notification should appear for non-verified teachers. However, it was showing up briefly for verified teachers. In the time between initial page load and setting the verification status of the teacher via Redux the teacher was considered unverified even though they were verified, we just didn't know it yet.  This happened because the verification status of the teacher was being checked in [progressRedux](https://github.com/code-dot-org/code-dot-org/blob/77cc4eb0cb69c92b5dc9caf6a55c3e6a9bc52c5f/apps/src/code-studio/progressRedux.js#L338) _after_ the notification warning had already loaded.  

**BEFORE**: Verified teachers would briefly see a non-applicable warning about locked resources, then the warning would disappear 
flash

To fix the issue I passed verification status from the server into the JS for the script overview page, where it could be used to dispatch verification status into Redux earlier in the loading of the page.  This is consistent with how these checks work for verification on the [course overview page](https://github.com/code-dot-org/code-dot-org/blob/77cc4eb0cb69c92b5dc9caf6a55c3e6a9bc52c5f/apps/src/sites/studio/pages/courses/show.js#L45). 

**AFTER**: Verified teachers don't briefly see the irrelevant warning 
no-flash

Non-verified teachers will still see the warning 
Screen Shot 2019-09-04 at 5 54 30 PM

NOTE: Follow up to #29321, in which I _thought_ the issue was fixed but either wasn't or has since regressed. Regardless, I removed `verificationCheckComplete` in favor of a new approach to solving the problem.

### CODE REVIEW

File: dashboard/app/views/scripts/show.html.haml [hunk 1]
```diff
@@ -1,6 +1,6 @@
--# This variable will get any data we need to pass along to scriptOverview.js
+-# This variable will get any data we need to pass along to scripts/show.js
 - script_data = @script.summarize(true, current_user)
```
Comment (Erin007) [inline dashboard/app/views/scripts/show.html.haml:2]: I changed the comment in this file based on #29509

File: apps/src/code-studio/verifiedTeacherRedux.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const SET_VERIFIED = 'verifiedTeacher/SET_VERIFIED';
 const SET_VERIFIED_RESOURCES = 'verifiedTeacher/SET_VERIFIED_RESOURCES';
 
-export const setVerified = () => ({type: SET_VERIFIED});
+export const setVerified = isVerified => ({type: SET_VERIFIED});
```
Comment (davidsbailey) [inline apps/src/code-studio/verifiedTeacherRedux.js]: remove unused `isVerified` param?

**Review by davidsbailey — APPROVED** [review]
Great solution @Erin007 , and great write up!

**Review by maddiedierker — APPROVED** [review]
🎉 great work!

**Pull request merged**